cg-spec/editors_draft.html

```diff
@@ -164,19 +164,18 @@ <h1>Introduction</h1>
     <section class="informative">
       <h2>Background and Motivation</h2>
 
-      <p>TODO, citing [[RDF-STAR-FOUNDATION]]</p>
+      <p>The RDF data model lets you state facts in three-part subject-predicate-object statements known as triples. For example, with a single RDF triple you can say that employee38 has a familyName of "Smith". A triple's predicate is a property specified with an IRI (an Internationalized version of a URI), identifying that property in a globally unambiguous way. A triple's subject and object can each be an IRI referencing any entity, and the object can also be a literal value such as "Smith" or data of other types such as dates, numbers, or Boolean values.</p>
+
+      <p>Sometimes, we want the subject or object of a triple to refer to another triple. For example, the statement "according to employee22, employee38 has a jobTitle of 'Assistant Designer'" can be modeled as a triple with "according to" as its predicate, employee22 as its object, and another triple as its subject, namely the triple "employee38 has a jobTitle of 'Assistant Designer'". This use of a triple as the subject or object resource of another triple so that we can say things about that triple is known as <a href='https://www.w3.org/TR/rdf11-mt/#reification'>reification</a>.
+      The concept of reification has always been part of RDF, but expressing it in RDF concrete syntaxes such as Turtle, N-Triples, and RDF/XML, as well as processing or querying it with SPARQL, has been verbose and cumbersome [[RDF-STAR-FOUNDATION]].</p>
```

Comment:
Here is the suggestion made by @rat10 in his [recent comment](https://github.com/w3c/rdf-star/pull/171#issuecomment-849140936) captured directly as a suggested change to this PR:

```suggestion
      The concept of reification has always been part of RDF [[RDF-STAR-FOUNDATION]], but expressing it in RDF concrete syntaxes such as Turtle and N-Triples, as well as processing or querying it with SPARQL, has been verbose and cumbersome. RDF/XML provides syntactic sugar for reification by means of an :ID attribute but, for other reasons, has become mostly a legacy format.</p>
```

Reply:
I agree to remove RDF/XML from this list, because indeed reification in this syntax is not as verbose as in others.

However, calling it a legacy format may sound to harsh to some people. It may also give the impression that an extension of RDF/XML to support RDF-star would be out of scope, which 1) is not the case, and 2) would [worry some people](https://lists.w3.org/Archives/Public/public-rdf-star/2021May/0079.html).

So I would rather just remove RDF/XML from the list, and may be add "most" in front of "concrete syntaxes" (

Reply:
> `but for other reasons has become mostly a legacy format.`

Remove this. Unrelated and opinion.